liquibase-standard/src/main/java/liquibase/change/core/CreateIndexChange.java

```diff
@@ -179,6 +182,11 @@ public Boolean getClustered() {
         return clustered;
     }
 
+    @DatabaseChangeProperty(description = "The index type to use. For example, 'btree' or 'gin'")
```

Comment:
Is this a breaking change for checksums?

Reply:
I added this new parameter to the excluded fields to prevent this down below in this same file -> https://github.com/liquibase/liquibase/pull/6901/files#diff-f1647dc293a46d72c22151fa77240ea144cc38788c2af645e71b1e74987e30a2R234